app/myfunctions.py

```diff
@@ -1,70 +1,66 @@
 def sort_numbers(num_list):
-
     class SortedNumbers:
-      def __init__(self, low=None, average=None, high=None):
-          self.low = low
-          self.average = average
-          self.high = high
+        def __init__(self, low=None, average=None, high=None):
```

Comment:
This is a huge commit with tons of changes, but they all have to do with the [PEP8](https://www.python.org/dev/peps/pep-0008/) guidelines. If you haven't read PEP8 I recommend it; Python is, much to its advantage, an opinionated and idiomatic language; there is generally speaking *one* correct way of formatting Python code, and that's PEP8. 

As such—there are linters, like [flake8](http://flake8.pycqa.org/en/latest/), which will automatically scan your code and highlight formatting errors. For instance, PEP8 specifies that all lines are indented by multiples of 4, so this indention is a violation.

And there are autoformatters, which will non-destructively autoformat your code to bring it in line with PEP8. [autopep8](https://pypi.python.org/pypi/autopep8) and [yapf](https://github.com/google/yapf) Are both quite good.